File: Other/ex1/main.exs
ans = File.read!("../input.txt") 
    |> String.split("\n\n") 
    |> Enum.map(fn x -> String.split(x, "\n") end) 
    |> Enum.map(fn x -> Enum.map(x, fn y -> String.to_integer(y) end) end) 
    |> Enum.map(fn x -> Enum.sum(x) end) 
    |> Enum.max()

IO.puts ans

File: Other/ex1/main.rb
file_path = '../input.txt'
file = File.open(file_path)
data = file.read.split("\n\n")

ans = 0
data.each do |x|
  calories = x.split("\n").map(&:to_i).sum
  ans = calories > ans ? calories : ans
end

puts ans